app/src/androidTest/java/com/hedvig/app/feature/offer/OfferScreen.kt

```diff
@@ -38,8 +38,8 @@ class OfferScreen : Screen<OfferScreen>() {
 }
 
 class ChangeDateSheet : Screen<ChangeDateSheet>() {
-    val autoSetDate = KButton { withId(R.id.autoSetDateText) }
-    val pickDate = KView { withId(R.id.datePickButton) }
+    val autoSetDate = KButton { withId(R.id.`@+id/auto_set_date_switch`) }
+    val pickDate = KView { withId(R.id.`@+id/date_pick_layout`) }
     val datePicker = KDatePickerDialog()
```

Comment:
Any ideas on how to interact with a MaterialDatePicker in kakao? I always seem to get stuck with these `Screen`s when refactoring existing code 🤔

Reply:
Could not find a way to select dates without creating the dialog with another builder, I rewrote the tests to check if the dialog is showing when pressing the date button. What I will do to test this instead is starting the bottom sheet fragment with a list of selected dates, the tests will not depend on the dialog picker in that case (and we will get state restoration!). What do you think?